packages/office-ui-fabric-react/src/components/Image/Image.tsx

```diff
@@ -127,7 +127,7 @@ export class Image extends BaseComponent<IImageProps, IImageState> {
   private _evaluateImage(): boolean {
     let { src } = this.props;
     let { loadState } = this.state;
-    let isLoaded = (src && this._imageElement.naturalWidth > 0 && this._imageElement.naturalHeight > 0);
+    let isLoaded = (src && this._imageElement.complete);
```

Comment:
I'm a bit nervous here. `complete` will be true even in a failure scenario. Do we want this? Because I think the behavior is changing. The previous behavior was to avoid showing broken link images. Now if you load http:/nonexistant.com/404_abc123.xyz, which 404s, you'll see a different result.